.github/ISSUE_TEMPLATE/bug_report_model.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug report related to our models
+description: Submit a bug report that's related to our models behavior
+title: '[BUG MODEL]: '
+labels: ['bug model', 'triage']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: model
+    attributes:
+      label: Model
+      description: What's the model?
+      options:
+        - open-mistral-7b
+        - open-mixtral-8x7b
+        - open-mixtral-8x22b
+        - mistral-small-latest
+        - mistral-medium-latest
+        - mistral-large-latest
+        - mistral-embed
+        - codestral-latest
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: model-payload
+    attributes:
+      label: Request Payload
+      description: Please provide the full request payload.
+      placeholder: If possible, if not provide as much information as you can such as "messages", "max_tokens", "temperature", "prefix", "tool_calls" and all kinds of information.
```

Comment:
If possible, much better for us if people give us repro with temperature=0 if they can manage it